.devcontainer/devcontainer.json

```diff
@@ -5,27 +5,38 @@
   "build": {
     "dockerfile": "Dockerfile"
   },
-
-  "settings": {},
-
-  // Add the IDs of extensions you want installed when the container is created.
-  "extensions": [
-    "dbaeumer.vscode-eslint",
-    "esbenp.prettier-vscode",
-    "ms-azuretools.vscode-docker",
-    "mhutchie.git-graph"
+  "mounts": [
+    "source=${localWorkspaceFolderBasename}-node_modules,target=${containerWorkspaceFolder}/node_modules,type=volume"
   ],
-
```

Comment:
I was showing a local linter warning for this, seems like the formatting/specification has since changed subsequently to initial rollout of the devcontainer when the project boilerplate was first initiated; I made corresponding update (i.e., nesting these properties into object `"customizations"."vscode": { ... }`) and that seemed to resolve the issue